### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by guillaumehussong** [npubkey/npubkey#1]
Angular dev and Nostr fanboy here ! 
I like your project.
Is it ok for you if I contribute ?
Cheers.

**Comment by npubkey** [comment]
Hi thanks for your interest!

Yes of course, I would be happy with contributions, there is about a million things left to do and improve haha.

If you just want to get involved and fix bugs / improve / add features you are free to. I could also share my current "To Do" list if you'd like

**Comment by guillaumehussong** [comment]
I am such a noob but really want to learn a lot.

Thanks you for having me in your mission !

Yes interested to see the ToDo list. Can we communicate via Element or others ?

Can I start by implementing a multi relays connections by default ? 

Cheers

**Comment by npubkey** [comment]
Yes I adding Relay Pools is on my To Do list, it would be great to have -- I am using nostr-tools library to do nostr stuff so you can see an example in the readme: https://github.com/nbd-wtf/nostr-tools

Whats is element?

**Comment by guillaumehussong** [comment]
Ok great. yes I saw that you are using nostr-tools which I know a little.

element: https://github.com/vector-im

**Comment by guillaumehussong** [comment]
better solution: npub1fhlwh03chgqj5fg9yvkr3r9f08ru70ne8f05qgccsa5ek3ytlw9s2r5vwy
